src/Helper/PaymentHelper.php

```diff
@@ -94,16 +111,28 @@ public function getAmountToPay(array $values, string $key): float {
   /**
    * Validates a given payment via the Nets Payment API.
    *
-   * @param string $endpoint
-   *   Nets Payment API endpoint.
-   *
-   * @return bool
-   *   Returns validation results.
+   * @param array<mixed> $element
+   *   Element array.
+   * @param \Drupal\Core\Form\FormStateInterface $formState
+   *   Form state object.
    */
-  public function validatePayment($endpoint): bool {
+  public function validatePayment(array &$element, FormStateInterface $formState): void {
+    // @FIXME: make error messages translateable.
```

Comment:
It would've been quicker to just wrap all messages in `$this->t(…)` than write this `@FIXME` … (see https://api.drupal.org/api/drupal/core%21lib%21Drupal%21Core%21StringTranslation%21StringTranslationTrait.php/trait/StringTranslationTrait/9).